### PR TITLE
post_features

### PR DESCRIPTION
**Pull request by ngocnt-3479**
## WHAT (optional)

- Post CRUD

## Evidence (Screenshot or Video)

image
image

### CODE REVIEW

**Comment by ngocnt-3479** [comment]
ready

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -5,20 +5,13 @@
...
     <%= render "layouts/header" %>
-    <div>
-      <% flash.each do |message_type, message| %>
-        <div class="alert alert-<%= message_type %>">
-          <strong><%= message %></strong>
-        </div>
-      <% end %>
-      <%= yield %>
-    </div>
+    <%= render "layouts/flash"%>
```
Comment (anlvq-1058) [inline app/views/layouts/application.html.erb:14]:
```suggestion
    <%= render "layouts/flash" %>
```

File: app/views/posts/_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<%= form_with model: post, id: id_name do |form| %>
+  <%= render "shared/error_messages", object: form.object %>
+  <div class="field">
+    <%= form.label :content%>
```
Comment (anlvq-1058) [inline app/views/posts/_form.html.erb:4]:
```suggestion
    <%= form.label :content %>
```

File: app/views/posts/_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    <%= form.text_area :content %>
+  </div>
+  <span class="image">
+    <%= form.label :image%>
```
Comment (anlvq-1058) [inline app/views/posts/_form.html.erb:8]:
```suggestion
    <%= form.label :image %>
```

File: app/views/shared/_user_info.html.erb [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<div class="main-box clearfix">
+  <h2><%= current_user.name %></h2>
+  <img src="https://bootdey.com/img/Content/avatar/avatar1.png" alt="" class="profile-img img-responsive center-block">
+  <div class="profile-since">
+      Member since: <%= current_user.created_at.strftime("%d %b %Y") %>
```
Comment (anlvq-1058) [inline app/views/shared/_user_info.html.erb:5]: "%d %b %Y" đưa vào file setting nhé

**Review by anlvq-1058 — COMMENTED** [review]
cmt

**Comment by ngocnt-3479** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]

**Comment by huongnt-2545** [comment]
merged

**Pull request merged**